package.json

```diff
@@ -50,7 +50,7 @@
     "custom-event-polyfill": "^1.0.6",
     "d3-selection": "2.0.0",
     "document-register-element": "1.13.1",
-    "generic-sequence-panel": "^1.2.5",
+    "generic-sequence-panel": "^1.3.0",
```

Comment:
Yes, it can go to production; I had been kind of sitting on it since I didn't know when things went out (and in my head it's linked to the change in JBrowse, but in reality they are completely separate).